docs/configuration.rst

```diff
@@ -0,0 +1,8 @@
+Configuration
+=============
+
+
+Configuration for the REANA system can be made through the file `config.yaml` under `templates`:
```

Comment:
Better use a language iike:

REANA-Resources-K8 default congifuration is defined in `templates/config.yaml` file. It is printed below for inspiration. You can define your own `myconfig.yaml` and then use `reana-resources-k8 build-manifests myconfig.yaml` to fit your computing cloud.  (*and link to CLI documentation*) 


```